presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestApproximateCountDistinct.java

```diff
@@ -169,7 +169,7 @@ private Page createPage(List<Object> values, double maxStandardError)
     /**
      * Produce a block with the given values in the last field.
      */
-    private static Block createBlock(Type type, List<Object> values)
+    protected static Block createBlock(Type type, List<Object> values)
```

Comment:
Retain private. Your subclass creates a block of specific type, which is quite straightforward.